eng/codecov.yml

```diff
@@ -0,0 +1,11 @@
+codecov:
+  branch: master
+  ci:
+    - dnceng.visualstudio.com
+
+coverage:
+  range: 60...90
+
+comment:
+  layout: "reach, diff, files"
```

Comment:
💡 `reach` is an impediment to mobile browsing and does not really provide a benefit. I recommend omitting `files` as well, though this can wait until some point in the future when you realize it's not especially helpful 😄